app/services/onlinetesting/OnlineTestExpiryService.scala

```diff
@@ -42,16 +43,13 @@ class OnlineTestExpiryServiceImpl(
   newHeaderCarrier: => HeaderCarrier
 )(implicit executor: ExecutionContext) extends OnlineTestExpiryService {
 
-  private final val ExpiredStatus = "ONLINE_TEST_EXPIRED"
   private implicit def headerCarrier = newHeaderCarrier
 
   override def processNextExpiredTest(): Future[Unit] = {
```

Comment:
You mean "nextExpiringApplication" like in the comment above right?
I assume so as it's the same method and also at this stage the application is not really expired yet.